groups/infrastructure/local.tf

```diff
@@ -0,0 +1,19 @@
+locals {
+  stack_name     = "utility-services"
+  stack_fullname = "${local.stack_name}-stack"
+  name_prefix    = "${local.stack_name}-${var.environment}"
+
+  stack_secrets              = jsondecode(data.vault_generic_secret.secrets.data_json)
+
+  application_subnet_pattern = local.stack_secrets["application_subnet_pattern"]
+  application_subnet_ids     = join(",", data.aws_subnets.application.ids)
+  kms_key_alias              = local.stack_secrets["kms_key_alias"]
+  vpc_name                   = local.stack_secrets["vpc_name"]
+  notify_topic_slack_endpoint = local.stack_secrets["notify_topic_slack_endpoint"]
+
+  ec2_ami_id = var.ec2_ami_id == "" ? data.aws_ami.ecs.id : var.ec2_ami_id
```

Comment:
Re: https://github.com/companieshouse/ocr-service-stack/pull/1/files#r1743727645

You've added this local but it's not being used anywhere - is it required?